### PR TITLE
feat(operator): introduce basic operator types 

### PR DESCRIPTION
**Pull request by 00nktk**
- Types for interaction with operator keypairs
- `eth_sign` implementation
- `eth_accounts` implementation

### CODE REVIEW

File: executor/Cargo.toml [hunk 1]
```diff
@@ -18,6 +18,7 @@ alloy-eips.workspace = true
 alloy-rlp.workspace = true
 
 anyhow.workspace = true
+clap.workspace = true
```
Comment (polachok) [inline executor/Cargo.toml:21]: 🤔

File: operator/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+use std::sync::Arc;
+
+use anyhow::bail;
+use clap::Args;
+use reth_primitives::Address;
+
+pub use operator::Operator;
+
+#[derive(Args)]
+pub struct Config {
```
Comment (polachok) [inline operator/src/lib.rs:15]: kinda weird to add cli args to libs don't you think?

**Review by polachok — APPROVED** [review]

File: operator/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    Err(error) => {
+                        tracing::warn!(?error, dir = ?path, "error reading directory entry");
+                        continue;
+                    }
+                }
+            }
+        }
+
+        let mut map = HashMap::new();
+        // TODO: tokio read_dir??
```
Comment (tukan) [inline operator/src/lib.rs:57]: We can remove this TODO. We run this code in the initialisation phase, so it makes little sense to make it async.

**Review by tukan — APPROVED** [review]

File: operator/src/lib.rs (hunk 2 above)
Comment (00nktk) [reply operator/src/lib.rs:15]: not as weird as having a 20+ field struct in `proxy` crate with no structure whatsoever IMO. 

I believe that defining a config layout is a responsibility of a lib.

File: operator/src/lib.rs (hunk 3 above)
Comment (00nktk) [reply operator/src/lib.rs:57]: I prefer for this TODO to stay since I plan on introducing reload logic that will be used in async code.

Comment (tukan) [reply operator/src/lib.rs:57]: Fair enough 👍

**Pull request merged**